spec/controllers/accounts_controller_spec.rb

```diff
@@ -32,6 +32,28 @@
       expect(response).to redirect_to teams_path
     end
 
+    it 'shows an error if you look into a team if youre admin and not member of' do
```

Comment:
Admin can look into a team, even if they're not member of it, can't they?
https://github.com/puzzle/cryptopus/wiki/Permissions
Team -> private -> Show Account